cpp/include/raft/neighbors/ivf_pq.cuh

```diff
@@ -74,6 +74,34 @@ inline auto build(raft::device_resources const& handle,
   return raft::spatial::knn::ivf_pq::detail::build(handle, params, dataset, n_rows, dim);
 }
 
+/**
```

Comment:
Can you please move the `defgroup` tag so that it only documents the mdspan-based APIs? we're deprecating the pointer-based APIs..